README.md

```diff
@@ -25,17 +25,12 @@ This package provides a KeyShot plugin script that creates jobs for AWS Deadline
 
 ### Using the KeyShot Submitter
 
-1. Install deadline and Pyside6
-    - e.g. `pip install deadline[gui]`
-2. Copy or link the file `deadline-cloud-for-keyshot/keyshot_script/Submit to AWS Deadline Cloud.py` to the KeyShot scripts folder.
-    - e.g. On Windows `C:/Users/<USER>/Documents/KeyShot/Scripts`
-3. Set the following environment variables
-    - Set the environment variable `DEADLINE_PYTHON` as the path to the Python installation where deadline-cloud and PySide2 were installed in step 1.
-      - e.g. On Windows if using Python 3.10 it might be `set DEADLINE_PYTHON=C:/Users/<USER>/AppData/Local/Programs/Python/Python310/python`
-    - Set the environment variable `DEADLINE_KEYSHOT` as the path to the `<PATH TO>/deadline-cloud-for-keyshot/src/deadline/keyshot_submitter` folder
-      - e.g. On Windows if the source was on the user's desktop it might be  `set DEADLINE_KEYSHOT=C:/Users/<USER>/Desktop/deadline-cloud-for-keyshot/src/deadline/keyshot_submitter`
-4. Launch KeyShot with the environment variables from step 3. set.
-5. The submitter can be launched within KeyShot from `Window > Scripting Console > Scripts > Submit to AWS Deadline Cloud > Run`
+1. Install `deadline[gui] >= 0.48.7,< 0.49` and `PySide6`
```

Comment:
nit: For the submitter I don't think we need to recommend pinning to < 0.49. We will have to update that every time and it's not required IMO.

Reply:
I'm confused. Didn't we want to pin based on an [earlier discussion](https://github.com/aws-deadline/deadline-cloud-for-keyshot/pull/93#discussion_r1655374540)?

Reply:
Spoke offline, and this can be punted until after merging. No sense holding this PR up more.